Documentation/op-guide/configuration.md

```diff
@@ -242,13 +242,13 @@ The security flags help to [build a secure etcd cluster][security].
 + default: false
 + env variable: ETCD_CLIENT_CERT_AUTH
 
-### --client-crl-file
+### --client-crt-file
```

Comment:
This flag should remain as --client-crl-file as this doesn't represent the client cert but rather represents the client certificate-revocation-list.